consensus/parlia/bohrFork.go

```diff
@@ -0,0 +1,96 @@
+package parlia
+
+import (
+	"context"
+	"errors"
+	"math/big"
+	mrand "math/rand"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/common/math"
+	"github.com/ethereum/go-ethereum/consensus"
+	"github.com/ethereum/go-ethereum/core/systemcontracts"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/internal/ethapi"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/params"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+func (p *Parlia) getTurnTerm(chain consensus.ChainHeaderReader, header *types.Header) (*uint8, error) {
+	if header.Number.Uint64()%p.config.Epoch != 0 ||
+		!p.chainConfig.IsBohr(header.Number, header.Time) {
+		return nil, nil
```

Comment:
just return 1 instead of nil? so that we can still use `uint8` instead of `*uint8` to make it simple

Reply:
accepted

Reply:
> just return 1 instead of nil? so that we can still use `uint8` instead of `*uint8` to make it simple

if change the return value type from `*uint8` to `uint8`

then in func `getTurnTerm`, will also return a meaningful value when error happens,
for example in the line 31 in func `getTurnTerm`
so return pointer is better IMO